README.md

```diff
@@ -66,3 +66,4 @@ We would like to thank these amazing open source projects and libraries that we
 | [github.com/react-bootstrap/react-bootstrap](https://github.com/react-bootstrap/react-bootstrap) | MIT License |
 | [github.com/sass/node-sass](https://github.com/sass/node-sass) | MIT License |
 | [loading.io/](https://loading.io/) | [Loading.io Free License](https://loading.io/license/#free-license) |
+| [Tone.js][https://tonejs.github.io]| [MIT License](https://github.com/Tonejs/Tone.js/blob/dev/LICENSE.md)|
```

Comment:
Please match the format of the lines for the other GitHub hosted projects:

```
| [github.com/Tonejs/Tone.js](https://github.com/Tonejs/Tone.js) | MIT License |
```

And move it up one line, between "github.com/sass/node-sass" and "loading.io".